README.md

```diff
@@ -1790,6 +1790,8 @@ IEEE Trans. Neural Netw. Learn. Syst. (TNNLS), 2021.  [Project page](https://sit
 - <a name="Wang23arxiva"></a>Wang, Z., Guo, J., Daniilidis, K.,  
 *[Un-EvMoSeg: Unsupervised Event-based Independent Motion Segmentation](https://arxiv.org/pdf/2312.00114.pdf)*,  
 arxiv, 2023.  [Project page](https://www.cis.upenn.edu/~ziyunw/un_evmoseg/).
+- <a name="Jiang24visapp"></a>Jiang, C., , J., Davoine, F. *[Event-based Semantic-aided Motion Segmentation](https://doi.org/10.5220/0012308100003660)*,  
+Proceedings of the 19th International Joint Conference on Computer Vision, Imaging and Computer Graphics Theory and Applications (VISAPP), 2024.
```

Comment:
Int. Joint Conf. on Computer Vision, Imaging and Computer Graphics Theory and Applications (VISAPP) 

(see other VISAPP 2020 entry)